telemetry integration support added (#3585)
GoogleCloudPlatform/magic-modules: third_party/terraform/website/docs/r/container_cluster.html.markdown

* `private_cluster_config` - (Optional) Configuration for [private clusters](https://cloud.google.com/kubernetes-engine/docs/how-to/private-clusters),
clusters with private nodes. Structure is documented below.

* `cluster_telemetry` - (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html)) Configuration for

>    [ClusterTelemetry](https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1beta1/projects.locations.clusters#Cluster.ClusterTelemetry) feature,

Reviewer: ```suggestion
   [ClusterTelemetry](https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1beta1/projects.locations.clusters#Cluster.ClusterTelemetry) feature.
```

Is there a GKE docs page that describes the feature in a more user-friendly way than the API page, or is that it for docs?

Author: I pointed to this page. This is the closest i see, https://cloud.google.com/monitoring/kubernetes-engine/installing#controlling_the_collection_of_application_logs

* `enable_intranode_visibility` - (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html))
    Whether Intra-node visibility is enabled for this cluster. This makes same node pod to pod traffic visible for VPC network.

> * `cluster_telemetry` - (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html))

Reviewer: Since this is a block, can you add a "structure is documented below" and then "the `cluster_telemetry` block supports" later on? you can see other places in this doc or in any of our generated docs where we do similar constructs.

Author: done

* `enable_intranode_visibility` - (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html))
    Whether Intra-node visibility is enabled for this cluster. This makes same node pod to pod traffic visible for VPC network.

The `cluster_telemetry` blocks supports

> * `type` - Telemetry integration for the cluster. Supported Values (`ENABLE, DISABLE, SYSTEM_ONLY`);

Reviewer: ```suggestion
* `type` - Telemetry integration for the cluster. Supported values (`ENABLE, DISABLE, SYSTEM_ONLY`);
```

>    `SYSTEM_ONLY` (Only system components are monitored and logged) available only on gke version 1.15 and later.

Reviewer: ```suggestion
   `SYSTEM_ONLY` (Only system components are monitored and logged) is only available in GKE versions 1.15 and later.
```

>    set `type = system_only` (Only system components are monitored and logged) available only on gke version 1.15 and later.

Author: done
